week2/homework/src/remove.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+function remove(todoTask) {
+  if (todoTask !== undefined) {
+    fs.readFile('./data.txt', 'utf-8', (err, data) => {
+      if (err) throw err;
+
+      const newValue = data.replace(todoTask, '');
+
+      fs.writeFile('./data.txt', newValue, 'utf-8', err => {
+        if (err) throw err;
+        console.log(`${todoTask} removed from the list`);
```

Comment:
what if the task is not in the list?

Reply:
Thanks I added another function to check if the task exists